x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/tail_sampling_settings.tsx

```diff
@@ -67,6 +70,24 @@ export function getTailSamplingSettings(): SettingsRow[] {
                 'Policies map trace events to a sample rate. Each policy must specify a sample rate. Trace events are matched to policies in the order specified. All policy conditions must be true for a trace event to match. Each policy list should conclude with a policy that only specifies a sample rate. This final policy is used to catch remaining trace events that don’t match a stricter policy.',
             }
           ),
+          helpText: (
```

Comment:
if `docsLinks` is undefined, shouldn't you avoid rendering the `helpText` in this case?

Reply:
That's a good point, I'll fix it